### PR TITLE
Theme deployment README update

### PR DESCRIPTION
**Pull request by elliottcrush**
#### What does this PR do? (please provide any background)
Updates the readme to describe the theme deployment functionality that now exists for the brand repository. Please check if it makes sense and / or needs anything adding.

#### What tests does this PR have?
None

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,4 +26,8 @@ In order to publish your local changes to NPM you'll need to do the following:
...
 3. Publish the branch and once your PR has been merged, [CircleCI](https://circleci.com/gh/holidayextras/brands) will automatically publish your changes to NPM.
-4. Change the application which references the individual brand to point to the new version.
\ No newline at end of file
+4. Change the application which references the individual brand to point to the new version.
+
+### Brand CSS Deployment
+
+Upon merge to master, [Circle](https://circleci.com/gh/holidayextras/brands) will now build a minfied CSS file for every `/brand/theme-deployment/` LESS file. This will be deploy a timestamped file to the [Brands S3 Bucket](http://s3-eu-west-1.amazonaws.com/hx-production-repo/brands) here. If you need to access the filename for the recently compiled and deployed theme you need to review the Circle CI logs for the recent merge. Here's an [example](https://circleci.com/gh/holidayextras/brands/114), scroll down to `$ ./scripts/deploy.sh` to view the filename.
```
Comment (stuartgiles) [inline README.md]: Is this a time stamp? Or a version number?

Comment (elliottcrush) [inline README.md]: Its a time stamp now.

**Comment by stuartgiles** [comment]
Discussed in person the benefits of timestamp over version number.
I'm happy with the reasoning +1

**Comment by jodiedoubleday** [comment]
taking :v:

**Comment by jodiedoubleday** [comment]
:shipit:

**Pull request merged**